pilot/pkg/bootstrap/server.go

```diff
@@ -405,7 +405,7 @@ func (s *Server) initMesh(args *PilotArgs) error {
 				s.mesh = meshConfig
 				if s.EnvoyXdsServer != nil {
 					s.EnvoyXdsServer.Env.Mesh = meshConfig
-					s.EnvoyXdsServer.ConfigUpdate(model.UpdateRequest{Full: true})
+					s.EnvoyXdsServer.ConfigUpdate(&model.PushRequest{Full: true})
```

Comment:
Does it need to be a reference ? It appears to be a simple bool, does it even need to be a struct ? Quite a few allocs...

Reply:
Its not just a bool in all places.. we have this struct so we can do incremental and partial (only push to some proxies) EDS pushes.